plugin/src/gatsby-node.ts

```diff
@@ -32,7 +32,7 @@ export function pluginOptionsSchema({ Joi }: PluginOptionsSchemaArgs) {
       .default(""),
     shopifyConnections: Joi.array()
       .default([])
-      .items(Joi.string().valid("orders", "collections")),
+      .items(Joi.string().valid("orders", "collections", "publications")),
```

Comment:
Maybe not the best way to opt-in. A lot depends on whether or not we want to source publication nodes.